src/OpenTelemetry.SemanticConventions/scripts/generate.ps1

```diff
@@ -0,0 +1,29 @@
+$SCRIPT_DIR = $PSScriptRoot
+$ROOT_DIR = "${SCRIPT_DIR}/../"
+
+# freeze the spec version to make SemanticAttributes generation reproducible
+$SPEC_VERSION = "1.25.0"
+$GENERATOR_VERSION = "latest"
+
+Set-Location $SCRIPT_DIR
+
+Remove-Item -r -fo semantic-conventions
+mkdir semantic-conventions
+Set-Location semantic-conventions
+
+git init
+git remote add origin https://github.com/open-telemetry/semantic-conventions.git
+git fetch origin v$SPEC_VERSION
+git reset --hard FETCH_HEAD
+Set-Location ${SCRIPT_DIR}
+
+docker run --rm `
+  -v ${SCRIPT_DIR}/semantic-conventions/model:/source `
+  -v ${SCRIPT_DIR}/templates:/templates `
+  -v ${ROOT_DIR}:/output `
```

Comment:
Since we are now generating many files, I think it would be better to drop them into a subfolder:

`-v ${ROOT_DIR}/Attributes/:/output`

Easier that way to blow them away and re-generate everything, if needed. Also it creates a nice separation between real files and auto-generated files.

Reply:
Done!